src/test/py_src/visit_test_main.py

```diff
@@ -1749,14 +1749,21 @@ def FilterTestText(inText, baseText, numdifftol):
                    baseWordT.count(".") == 2 and baseWordT.endswith("."):
                     inWordT = inWordT.rstrip(".")
                     baseWordT = baseWordT.rstrip(".")
-                inStart = tmpText.find(inWords[w], inStart)
+                if (sys.version_info > (3, 0)):
+                    inStart = tmpText.find(inWords[w], inStart)
+                else:
```

Comment:
I didn't realize we still supported Python 2.

Reply:
There are several other code blocks in that file that condition on py 2/3 so I thought it was still necessary.